locales/zh_CN.yml

```diff
@@ -0,0 +1,59 @@
+manifest:
+  name: "密码生成器"
+  description: "密码生成器扩展可让您一键轻松生成长而安全的密码"
+  author: "Eugene Fox"
+
+common:
+  reset: "重置"
+  copy: "复制"
+
+errors:
+  too_short: "密码的长度必须至少为4个字符"
+  no_characters: "必须至少选择一组字符"
+  too_long: "长度太长，无法生成具有唯一字符的密码"
+
+about:
+  title: "关于"
+  developed_by: "由 Eugene Fox 开发"
+  lincensed_under: "协议："
+  mit_license: "MIT License"
+  translation_cta:
+    text: "发现错别字或需要翻译成您的语言？"
+    button: "从这里开始"
+  links:
+    website: "我的网站"
+    source: "源代码"
+    changelog: "更新日志"
+  cta:
+    feedback: "提供反馈"
+    sponsor: "Buy me a coffee"
```

Comment:
I think this line should be translated as well. But since "Buy me a coffee" is basically both CTA and the name of the service, it should be translated into either "Support me" or "Donate"